umbral/pre.py

```diff
@@ -507,10 +470,19 @@ def _open_capsule(capsule: Capsule, bob_private_key: UmbralPrivateKey,
     """
     params = params if params is not None else default_params()
 
-    priv_b = bob_private_key.bn_key
-    pub_b = priv_b * params.g
+    priv_b = bob_privkey.bn_key
+    pub_b = bob_privkey.get_pubkey().point_key
```

Comment:
If my PR gets in before this one, you can do `bob_privkey.pubkey.point_key` because it gets cached.